tests/integration/states/test_ssh_auth.py

```diff
@@ -12,6 +12,8 @@
 from tests.support.mixins import SaltReturnAssertsMixin
 from tests.support.runtests import RUNTIME_VARS
 
+KEY = "AAAAC3NzaC1lZDI1NTE5AAAAIEmY2ZOSneFGN+b7+S9eLPGOqPF2yl9AgTCkZHuWKjC5"
```

Comment:
Isn't it bad to create a global variable in the test suite

Reply:
Maybe it's not very pytest-ee. Though, the alternative of making a fixture also creates a global variable. I.E. Function definitions are also variables.